README.md

```diff
@@ -1,4 +1,4 @@
-## test-results-analyzer
+## [Test Results Analyzer](https://plugins.jenkins.io/test-results-analyzer/)
```

Comment:
I like the idea of hyperlinking from the title to the plugins page.  Nice improvement.
```suggestion
# [Test Results Analyzer](https://plugins.jenkins.io/test-results-analyzer/)
```